core/drivers/rstctrl/stm32_rstctrl.h

```diff
@@ -9,12 +9,24 @@
 struct stm32_rstline {
 	unsigned int id;
 	struct rstctrl rstctrl;
+	const struct stm32_reset_data *data;
 	SLIST_ENTRY(stm32_rstline)link;
 };
 
 struct stm32_rstline *to_rstline(struct rstctrl *rstctrl);
 
+struct stm32_reset_cfg {
```

Comment:
```c
/*
 * struct stm32_reset_cfg - Reset line controller data
 * @offset: Byte offset in reset controller IOMEM
 * @bit_idx: Bit position of reset line control at IOMEM @offset
 * @set_clr: True is @offset is an atomic SET/CLR register, false otherwise
 * @inverted: True is reset line is asserted at level 0, false otherwise
 * @no_deassert: True is reset line cannot be deasserted, false otherwise
 * @no_timeout: True if reset state cannot be read back for timeout detection
 */